hedera-node/src/test/resources/fees/feeSchedules.json

```diff
@@ -2,7 +2,7 @@
   "currentFeeSchedule" : [ {
```

Comment:
I guess that once we add unit tests, we should be modifying this json file?

Reply:
yes